core/camel-core-model/src/main/java/org/apache/camel/model/rest/VerbDefinition.java

```diff
@@ -141,6 +144,22 @@ public void addOutput(ProcessorDefinition<?> processorDefinition) {
         route.addOutput(processorDefinition);
     }
 
+    public boolean isDeprecated() {
```

Comment:
The getter/setter should generally be same type, there can be a Boolean getDeprecated

Reply:
You're right, I'm sorry I did a quick refactoring due to issues with xml generation, I had an error in ModelXmlParserGeneratorMojo with boolean type, but forgot about the getter, from camel codebase I see that Boolean getter are both get* and is*, is there any preference?

Reply:
For Boolean then get is preferred